README.rst

```diff
@@ -37,6 +37,7 @@ numpy            1.9.0
 nibabel          1.4.0dev
 pydicom          0.9.9
 dcmstack         0.7.0dev
+pymongo          2.8
```

Comment:
I have no idea if this is the correct version, but I had to install some version in order for `bson` to import correctly (and I tried `pip install bson` first, which didn't work since it needs the pymongo `bson` package...)